niworkflows/interfaces/segmentation.py

```diff
@@ -36,7 +36,7 @@ def _post_run_hook(self, runtime):
         overlaid '''
         self._anat_file = self.inputs.in_files[0],
         self._mask_file = self.aggregate_outputs().tissue_class_map
-        self._seg_files = self.aggregate_outputs().tissue_class_files
+        self._seg_files = self.aggregate_outputs().tissue_class_files[1:]
```

Comment:
This is the CSF class which in combination with other classes was showing skullstripping mask. I've added a comment explaining this.